build.py

```diff
@@ -1555,6 +1555,11 @@ def core_build(cmake_script, repo_dir, cmake_dir, build_dir, install_dir,
         cmake_script.cp(
             os.path.join(repo_install_dir, 'lib', 'libtritonserver.so'),
             os.path.join(install_dir, 'lib'))
+        # Copy boost filesystem library used to compile tritonserver
+        cmake_script.mkdir(os.path.join(install_dir, 'boost_filesystem'))
+        cmake_script.cp(
+            os.path.join('/usr', 'lib', 'libboost_filesystem.so.1.80.0'),
+            os.path.join(install_dir, 'boost_filesystem'))
```

Comment:
What is this `install/boost_filesystem` dir used for? So it gets packaged up in jetson tarball?

Reply:
Exactly. It is just a way to package the version of libboost_filesystem.so.1.80.0 used to compile tritonserver into the final tarball. From there users can add it to `/usr/lib` or a custom location and add it to their linker path.